src/ElectronNET.API/ElectronNET.API.csproj

```diff
@@ -28,8 +28,8 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
     <PackageReference Include="SocketIOClient" Version="3.1.2" />
-    <PackageReference Include="System.Drawing.Common" Version="8.0.16" />
-    <PackageReference Include="System.Text.Json" Version="8.0.5" />
+    <PackageReference Include="System.Drawing.Common" Version="10.0.1" />
```

Comment:
How would that work if you only have .NET 8 ? I think we need to reference the lower ones here.

Reply:
But System.Text.Json cannot be dropped. It's not included in net6.